modules/installation-azure-config-yaml.adoc

```diff
@@ -195,73 +225,93 @@ If you disable simultaneous multithreading, ensure that your capacity planning a
 <14> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
 endif::openshift-origin[]
 endif::vnet[]
 ifdef::private[]
-ifndef::openshift-origin[]
+ifndef::openshift-origin,azure-restricted[]
 <15> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
 +
 [IMPORTANT]
 ====
 The use of FIPS Validated / Modules in Process cryptographic libraries is only supported on {product-title} deployments on the `x86_64` architecture.
 ====
 <16> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
-endif::openshift-origin[]
+endif::openshift-origin,azure-restricted[]
 ifdef::openshift-origin[]
 <15> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
 endif::openshift-origin[]
 endif::private[]
 ifdef::gov[]
-ifndef::openshift-origin[]
+ifndef::openshift-origin,azure-restricted[]
 <16> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
 +
 [IMPORTANT]
 ====
 The use of FIPS Validated / Modules in Process cryptographic libraries is only supported on {product-title} deployments on the `x86_64` architecture.
 ====
 <17> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
-endif::openshift-origin[]
+endif::openshift-origin,azure-restricted[]
 ifdef::openshift-origin[]
 <16> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
 endif::openshift-origin[]
 endif::gov[]
 ifndef::vnet,private,gov[]
-ifndef::openshift-origin[]
+ifndef::openshift-origin,azure-restricted[]
 <10> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
 +
 [IMPORTANT]
 ====
 The use of FIPS Validated / Modules in Process cryptographic libraries is only supported on {product-title} deployments on the `x86_64` architecture.
 ====
 <11> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
-endif::openshift-origin[]
+endif::openshift-origin,azure-restricted[]
+ifndef::azure-restricted[]
 ifdef::openshift-origin[]
 <10> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
 endif::openshift-origin[]
+endif::azure-restricted[]
 endif::vnet,private,gov[]
 +
 [NOTE]
```

Comment:
This Note should not be displayed again for Azure restricted network. It is already displayed above.

Reply:
This Note is still getting displayed twice. To rectify the issue you can move the following lines after this Note and remove lines 244 to 248:
```
<17> Provide the contents of the certificate file that you used for your mirror registry.
<18> Provide the `imageContentSources` section from the output of the command to mirror the repository.
<19> How to publish the user-facing endpoints of your cluster. When using Azure Firewall to restrict Internet access, set `publish` to `Internal` to deploy a private cluster. The user-facing endpoints then cannot be accessed from the internet. The default value is `External`.
```